Makefile

```diff
@@ -44,6 +50,13 @@ test-cpp-asan: TOSCA_CPP_BUILD = Debug
 test-cpp-asan: TOSCA_CPP_ASAN = ON
 test-cpp-asan: test-cpp
 
+test-cpp-coverage: TOSCA_CPP_BUILD = Debug
+test-cpp-coverage: TOSCA_CPP_COVERAGE = ON
+test-cpp-coverage: test-cpp
```

Comment:
I am terrible with Makefiles, but isn't this dependency on `test-cpp` cause the tests to be run twice?

Reply:
I haven't written a makefile in 20 years... yes, the unit test was running twice.